### PR TITLE
fix Issue5847 as required（make the icon always visible and just adjust the count accordingly）

### LINKED ISSUE #5847: [Bug]: Upload icon vanishes at times even when there are uploads pending to complete
### Summary

The upload queue icon that we now have with latest app revamp seems to vanish at times even though there are uploading pending to be completed. It makes it difficult to manage the ongoing uploads.

If we couldn't figure out what's causing this, I suppose we could at least do a stopgap solution and just show the upload queue icon all the time to avoid confusion.

### Steps to reproduce

1. Initiate some uploads in the app (I initiated the upload by sharing the image to the app)
2. Complete the details necessary for the upload.
3. Go to the Contributions screen of the app.

### Expected behaviour

The upload queue should always be visible making it easy to manage ongoing uploads.

### Actual behaviour

The upload queue icon flickers and is sometimes not visible making it difficult to access the queue and manage ongoing uploads.

### Device name

OnePlus Nord

### Android version

Android 12

### Commons app version

main

### Device logs

_No response_

### Screen-shots

- [Screen record showing the icon flcikering](https://drive.google.com/file/d/15XPLcJ-6nZIF6ISHY3_rJ8E9D84XwO3L/view?usp=drive_link)
- [Screen record showing the icon only appearing after an app reopen](https://drive.google.com/file/d/12lD7vPH2Uj2NYMwgbhNo59Oa2MvgKeFJ/view?usp=drive_link)

### Would you like to work on the issue?

None

### PR DESCRIPTION
**Pull request by bxy379987**
comment out the setVisibility method and all calls to setVisibility method， and make pending_upload_icon always visible

**Description (required)**

Fixes #5847

What changes did you make and why?
Comment out the original setVisibility method and related methods, then set pending_upload_icon to visible, to ensure that the pending_upload_icon is always visible on the contribution interface, making it convenient for users

**Tests performed (required)**
the upload_icon is always visible

Tested {build variant, e.g. ProdDebug} on {Pixel 8} with API level {API 34}.

**Screenshots (for UI changes only)**
image

Need help? See https://support.google.com/android/answer/9075928

---

_Note: Please ensure that you have read CONTRIBUTING.md if this is your first pull request._

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java [hunk 1]
```diff
@@ -310,13 +310,13 @@ public void setNotificationCount() {
...
-    public void setUploadIconVisibility() {
-        contributionController.getFailedAndPendingContributions();
-        contributionController.failedAndPendingContributionList.observe(getViewLifecycleOwner(),
-            list -> {
-                updateUploadIcon(list.size());
-            });
-    }
+//    public void setUploadIconVisibility() {
+//        contributionController.getFailedAndPendingContributions();
+//        contributionController.failedAndPendingContributionList.observe(getViewLifecycleOwner(),
```
Comment (sivaraam) [inline app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java:315]: I think we could also comment the `failedAndPendingContributionList` member and the corresponding `getFailedAndPendingContributions` as I suppose they are not used for any other purpose.

**Review by sivaraam — COMMENTED** [review]
Thank you for the quick fix! Just some minor comments and it is good to go!

**Comment by nicolas-raoul** [comment]
@bxy379987 Any chance to address Sivaraam's comments above? Thanks! :-)

**Comment by bxy379987** [comment]
Ok sir, sorry, I was taking four exam this two weeks, I forgot to continue , I will improve it again before the end of this week

**Comment by sivaraam** [comment]
@bxy379987 would you possibly be able to continue working on this change and address Nicolas' comment? We need this MR before we proceed to the production release of v5.1.0.

It's fine if you're occupied with other work, we could take it from here 🙂

**Comment by bxy379987** [comment]
I am sorry sir, I am a Asian student and this I need to take my country’s civil servant exam during this vacation, which is very important to me. I sincerely apologize for not meeting expectations in my previous work and for wasting your time. I didn't even see Nicolas’s previous response,  and I'm really sorry about that. But at this stage, I truly don’t have the time or energy to continue, sorry sir, thank you for giving me this oppotunity to participate your program.

**Comment by nicolas-raoul** [comment]
@bxy379987 No problem at all, thanks for your contribution, and we wish you the best for your exams! :-)

**Comment by nicolas-raoul** [comment]
The feature is now part of the app.
Thanks all! :-)

**Pull request closed without merging**